optimum/exporters/tasks.py

```diff
@@ -692,6 +692,17 @@ class TasksManager:
             "text-classification",
             onnx="GemmaOnnxConfig",
         ),
+        "git": supported_tasks_mapping(
+            "feature-extraction",
+            "image-text-to-text",
+            "image-to-text",
+            onnx="GITOnnxConfig",
+        ),
+        "git-vision-model": supported_tasks_mapping(
+            "feature-extraction",
+            "image-to-text",
+            onnx="GITVisionModelOnnxConfig",
+        ),        
```

Comment:
Removed in https://github.com/marcindulak/optimum/commit/ea2321cd68cd63668454a2958e5fc923bb5e8950

I imagined that `git-vision-model` is expected since there is a separate `clip-vision-model`.
The docs show CLIPVisionModel
https://huggingface.co/docs/transformers/main/model_doc/clip#transformers.CLIPVisionModel
and GitVisionModel
https://huggingface.co/docs/transformers/main/model_doc/git#transformers.GitVisionModel
so I thought the setup will be similar.

I see `git_vision_model` in https://huggingface.co/microsoft/git-large/blob/main/config.json, but it's nested under `vision_config`. Is this the reason why there is no separate OnnxConfig needed?